pkg/client/group_mgmt_client.go

```diff
@@ -137,7 +137,7 @@ func (client *GroupMgmtClient) Put(path, id string, payload interface{}) (interf
 		}
 		return nil, fmt.Errorf("http response error: status (%d), messages: %v", response.StatusCode(), errResp)
 	}
-	return unwrap(response.Body(), payload)
+	return unwrap(response.Body(), respHolder)
 }
 
 // Get : Only used to get a single object with the given ID
```

Comment:
Rename intf to respHolder for consistency?

Reply:
Sure, next PR.